script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -101,7 +101,7 @@ I have lots of questions, but I don't know anything about Hisui and Kohaku-san a
 [sha:bb3f2a66b35f506286ce01a5935954102dff6c30]{
 -- Page 527, Offset 2497.
 -- 「……お世話って、やっぱりそういうコトか」
-"...Taking care, you say. That's what I thought."
+"Attending to? I guess it's just as I thought..."
```

Comment:
Doesn't quite feel right, but I'm tired.
It's like "so it's really does come down to this" or something.